src/api/pack/xdsl/move/pack-xdsl-move.v6.service.js

```diff
@@ -12,6 +12,11 @@ angular.module('ovh-api-services').service('OvhApiPackXdslMoveV6', ($resource, P
       url: '/pack/xdsl/:packName/addressMove/moveOffer',
       isArray: false,
     },
+    subserviceToDelete: {
```

Comment:
Action name now differ from the endpoint.

**suggesion:** Could you please rename it as:
```suggestion
    servicesToDelete: {
```